### DISCUSSION TITLE
How to create an order, query order information, etc., from java code

### DISCUSSION THREAD

**Discussion by CSXYA** [openTCS/opentcs#83] [Q&A]
What should I do, please,To create, query, and cancel shipping orders in java code,And changing the map in the kernel?

**Comment by swltr** [comment]
Please check the [developer's guide](https://www.opentcs.org/docs/5.8/developer/developers-guide/opentcs-developers-guide.html) first.

**Reply by CSXYA** [reply]
![image](https://user-images.githubusercontent.com/124949134/226222150-87d75632-7b07-461f-9b7e-108b09605887.png)
api has these two methods, how do I get these two methods?

**Reply by swltr** [reply]
This method is meant as a placeholder. Please see [the developer's guide](https://www.opentcs.org/docs/5.8/developer/developers-guide/opentcs-developers-guide.html#_acquiring_service_objects) on how to acquire service objects.
